toonz/sources/toonzlib/tstageobject.cpp

```diff
@@ -385,11 +385,11 @@ TStageObject::TStageObject(TStageObjectTree *tree, TStageObjectId id)
     , m_ikflag(0)
     , m_groupSelector(-1) {
   // NOTA: per le unita' di misura controlla anche tooloptions.cpp
-  m_x->setName("W_EW");
+  m_x->setName("X");
```

Comment:
At my first attempt I did that - replaced "W_EW" to  "W_X". 
And did the same replacement in "stuff/config/current.txt" - 
https://github.com/opentoonz/opentoonz/pull/3340/files#diff-6f4b6e30061aea387a274435ce91f8c8R819-R820

But when I launched OpenToonz, it displayed "W_X" instead of "X" in Function Editor.
This is because I haven't copied updated version of "stuff/config/current.txt" to my stuff directory. So, I had old "current.txt" file, which has no line "W_X" replacement string.

So, I wasn't sure how to deal with this. How we will make sure that "stuff/config/current.txt" is updated?

This actually reminds me old issue - https://github.com/opentoonz/opentoonz/issues/1375

Should I make a fix to ship some "config.txt" with application, which will set default values?

Reply:
If you update the `stuff/config/current.txt` file, when users reinstall on top of existing installations as an upgrade, that will overwrite it automatically so you shouldn't have to supply any extra files for conversions.  It will just happen.

Although Windows users do have the ability to skip the stuff folder overwrite during an upgrade, I don't recommend that to anyone because you will not get any changes to config files like this and that can lead to potential issues and crashes if not handled correctly.

Reply:
@manongjohn Thank you for clarification, I have made required changes. ^__^

Still, I am thinking about following: if the `stuff/config/current.txt` file is a subject of overwrite on every update of OT, then it would be better to include it into OpenToonz distro (`share/opentoonz`?) and remove from user config. 

We just need to change a single line - https://github.com/opentoonz/opentoonz/blob/22ad3653a86cb7281fcc7b47937dcbaaebe15801/toonz/sources/tnzbase/stringtable.cpp#L60
...and modify CMake rules to install into proper dir.

@shun-iwasawa @manongjohn what do you think about that?

Reply:
`current.txt` consists of mainly the UI name definitions of fxs and their parameters. IMO it should be united into the translation files in the future development.
(We need some trick to do so since the UI names are also used as the expression syntax, though.)

Reply:
@shun-iwasawa 
it would be just great to have the ability to translate fx´s!

Reply:
@shun-iwasawa I have found that `menubarcommandids.h` does a similar job as `current.txt`. As temporary solution we can merge `current.txt` into code in similar way, I guess?